api/integrations/github/github.py

```diff
@@ -55,18 +57,28 @@ def tag_feature_per_github_event(
         | Q(external_resources__type="GITHUB_ISSUE"),
         external_resources__url=metadata.get("html_url"),
     ).first()
+    repo: list[str] = repo_full_name.split("/")
+    tagging_enabled = (
+        GitHubRepository.objects.filter(
+            project=feature.project, repository_owner=repo[0], repository_name=repo[1]
+        )
+        .first()
+        .tagging_enabled
+    )
```

Comment:
`.filter `was changed to `.get`.
It should not raise an AttributeError because this function is only triggered when the webhook is called, which only happens when there is an integration with the issues/PRs of a GitHub repo with Flagsmith. Therefore, using .get should be appropriate in this context.